packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js

```diff
@@ -13,6 +13,8 @@ export default class InMeetingActions {
   namespace = MEETINGS;
 
   constructor() {
```

Comment:
if you want to convert the class to ts its the right time to do :D

Reply:
done